README.md

```diff
@@ -40,6 +52,14 @@ Optionally, if you want to use the task planning skill provided in the standard
 
 When asked for the install folder, you can insert a location of your preference or just leave the default.
 
+Build with catkin
+> cd /path/to/your/catkin_ws
```

Comment:
Missing newlines

Reply:
Solved in https://github.com/RVMI/skiros2/pull/40/commits/aa1803cec49cd7e11d4ed5888afe81d81a05d345